lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,22 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(opts, callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    addComputation(opts, requestId, transports.WEBSOCKET, callbacks, requestType, isRetryPatchMode);
```

Comment:
@mpetazzoni changed it to this, but can requestType and isRetryPatchMode be undefined?

Reply:
var requiredParams = { execute: ['program'], explain: ['incidentId'] };

Reply:
seems like these ^ are required